source/configuration.txt

```diff
@@ -249,15 +249,40 @@ Partitioner Configuration
        **Default:** ``_id``
 
    * - ``partitionSizeMB``
-     - The size (in MB) for each partition
+     - The size (in MB) for each partition. Smaller partition sizes 
+       create more partitions containing fewer documents.
 
        **Default:** ``64``
 
    * - ``samplesPerPartition``
-     - The number of sample documents to take for each partition.
+     - The number of sample documents to take for each partition in 
+       order to establish a ``partitionKey`` range for each partition. 
+       A greater number of ``samplesPerPartition`` increases the 
+       precision of finding partition ranges that match the 
```

Comment:
Yes, 'precision' is the wrong word in this sentence - I don't want to use chance or probability because it makes it sound strangely risky when this is just fine-tuning. 

Went with: "A greater number of ``samplesPerPartition`` helps to find ``partitionKey`` ranges that more closely match the ``partitionSizeMB`` you specify."